index.html

```diff
@@ -0,0 +1,108 @@
+            <p class=site-title>maxwell newton</p>
+        </div>
+
+        <div class="nav-menu"> <!-- right menu-->
+
+            <div class="nav-menu-item"> <!-- LinkedIn button -->
+                <a href="https://www.linkedin.com/in/maxwellnewton/">linkedin</a>
+            </div>
+
+            <div class="nav-menu-item"> <!-- GitHub button -->
+                <a href="https://github.com/maxwell-01">github</a>
+            </div>
+
+        </div>
+
+    </nav>
+
+    <header> <!-- flex parent for heading -->
+
+        <h1>Technologist and<br>Software Engineer<br>based in Bristol</h1>
+
+    </header>
+
+    <main>
+
+        <section class="intro-section"> <!-- introduction section, flex parent -->
+
+            <div> <!-- left side text -->
+                <h2>Hi, I'm Max.</h2>
+                <hr>
+                <p class="intro-section-message">My dream is to combine professional technology and engineering skills to help build a future that’s sustainable and allows people to spend time doing what’s most important to them.</p>
+
+            </div>
+
+            <div> <!-- right side image -->
+                <img class="intro-section-image" src="maxwell-newton-profile-picture.jpeg" alt="A profile picture of Maxwell Newton out on a hike">
+
+            </div>
+
+        </section>
+
+        <section class="timeline-section">
+            <h2>A Brief Timeline Of Me</h2>
+            <div class="timeline-parent">
+                <h3>1989</h3>
+                <p>I arrive!</p>
+                <h3>2005</h3>
+                <p>Built my first computer</p>
+                <h3>2011</h3>
+                <p>Started my own business building websites</p>
+                <h3>2013</h3>
+                <p>Graduated brunel university with 1st class honours in Mechanical Engineering<br>
+                    Started my first job at digiterre</p>
+                <h3>2015</h3>
+                <p>Completed the Edinburgh half marathon in 1hr 28mins</p>
+                <h3>2016</h3>
+                <p>Drove for 6 weeks across Europe & Asia on the Mongol Rally<br>
+                    Completed the Rob Roy biathlon in the Scottish highlands, finished first in class</p>
+                <h3>2017</h3>
+                <p>Moved to Cardiff and joined EPC</p>
+                <h3>2018</h3>
+                <p>Bought my first house in Bristol<br>
+                    Learnt python basics and wrote my first script</p>
+                <h3>2019</h3>
+                <p>Delivered €5.5m automotive contract for cutting-edge truck turbo component<br>
+                    Travelled around the world, grew a beard & got engaged</p>
+                <h3>2020</h3>
+                <p>Started at Roqqett as Head Of Operations</p>
+                <h3>2021</h3>
+                <p>Moved and started renovating an old house</p>
+                <h3>2022</h3>
+                <p>Started at iO Academy.... and more!</p>
```

Comment:
better to wrap each pair of year + thing into a separate div, that way they will be easier to position on the page